cache_images/fedora_packaging.sh

```diff
@@ -152,22 +144,19 @@ else
 fi
 
 
+# Packages needed for some special testing cases
+# where the test will perform the installation.
```

Comment:
Could I kindly ask for a maintainer-friendly comment here? Something like "we download the rpm for these packages, but don't install them; tests wishing to use them can install them by running dnf install /path/to/where/the/rpm/was/downloaded"

Reply:
I surely hope all possible future maintainers know `package==rpm` and use `dnf` to install them (similar for Ubuntu) :rofl:  I've got a ***bunch** of testing to do with these images once they build, so I'd prefer to prioritize that over a comment verbosity fix.  Though if I need to re-roll the images again here for some reason, I'm happy to copy-paste your suggestion in.

Reply:
What I meant was, it is not clear where the downloads go. But OK, a careful reader can figure it out. Once you get that flake fixed, feel free to merge.

Reply:
You're in luck, something else broke.  Adding your suggested comment...